js/src/admin/compat.js

```diff
@@ -1,17 +1,21 @@
 import compat from '../common/compat';
 
 import saveSettings from './utils/saveSettings';
+import ExtensionData from './utils/ExtensionData';
+import extensionEnabled from './utils/extensionEnabled';
```

Comment:
I feel like this file should be called `isExtensionEnabled` - I had to go to the file to see what it did.